Tools/BatchScripts/batch_summit.sh

```diff
@@ -32,6 +32,9 @@ export OMPI_MCA_coll_ibm_skip_barrier=true
 #export FI_OFI_RXM_USE_SRX=1  # libfabric: use shared receive context from MSG provider
 
 # OpenMP: one thread per MPI rank and GPU
+# ROMIO has a hint for GPFS named IBM_largeblock_io which optimizes I/O with operations on large blocks
+export IBM_largeblock_io=true
+
```

Comment:
I would add this here and for power9 CPUs below, which automates the process completely.
I found the `LSB_HOSTS` environment variable [here](https://www.bsc.es/support/LSF/9.1.2/lsf_config_ref/index.htm?lsf_envars_job_exec.html~main), quickly checked and this works :)
```suggestion

# MPI-I/O: ROMIO hints for parallel HDF5 performance
export OMPI_MCA_io=romio321
export ROMIO_HINTS=./romio-hints
#   number of hosts: unique node names minus batch node
NUM_HOSTS=$(( $(echo $LSB_HOSTS | tr ' ' '\n' | uniq | wc -l) - 1 ))
cat > romio-hints << EOL
   romio_cb_write enable
   romio_ds_write enable
   cb_buffer_size 16777216
   cb_nodes ${NUM_HOSTS}
   EOL

```

Reply:
Awesome!